poc/three/load_students.py

```diff
@@ -0,0 +1,274 @@
+                return False
+
+        except requests.exceptions.RequestException as e:
+            print(f"Authentication failed: {e}")
+            return False
+
+    def create_student(self, student_data: Dict) -> Optional[str]:
+        """Create a student record in Ed-Fi API"""
+        url = f"{self.base_url}/ed-fi/students"
+
+        try:
+            response = self.session.post(url, json=student_data)
+
+            if response.status_code == 201:
+                # Student created successfully
+                location = response.headers.get('Location', '')
+                student_id = location.split('/')[-1] if location else None
+                print(f"✓ Created student: {student_data['studentUniqueId']}")
+                return student_id
+            elif response.status_code == 409:
+                # Student already exists
+                print(f"⚠ Student already exists: {student_data['studentUniqueId']}")
+                return "existing"
+            else:
+                student_id = student_data['studentUniqueId']
+                status = response.status_code
+                text = response.text
+                print(f"✗ Failed to create student {student_id}: {status} - {text}")
+                return None
+
+        except requests.exceptions.RequestException as e:
+            print(f"✗ Error creating student {student_data['studentUniqueId']}: {e}")
+            return None
+
+    def create_student_school_association(self, association_data: Dict) -> Optional[str]:
+        """Create a student school association (enrollment) record in Ed-Fi API"""
+        url = f"{self.base_url}/ed-fi/studentSchoolAssociations"
+
+        try:
+            response = self.session.post(url, json=association_data)
+
+            if response.status_code == 201:
+                # Association created successfully
+                location = response.headers.get('Location', '')
+                association_id = location.split('/')[-1] if location else None
+                student_id = association_data['studentReference']['studentUniqueId']
+                school_id = association_data['schoolReference']['schoolId']
+                print(f"✓ Created enrollment for student {student_id} at school {school_id}")
+                return association_id
+            elif response.status_code == 409:
+                # Association already exists
+                student_id = association_data['studentReference']['studentUniqueId']
+                school_id = association_data['schoolReference']['schoolId']
+                print(f"⚠ Enrollment already exists for student {student_id} at school {school_id}")
+                return "existing"
+            else:
+                student_id = association_data['studentReference']['studentUniqueId']
+                status = response.status_code
+                text = response.text
+                print(f"✗ Failed to create enrollment for student {student_id}: {status} - {text}")
```

Comment:
It might be nice if the application understood that the response is in JSON format, and perhaps should convert to a single line string before logging.